members/M001176.yaml

```diff
@@ -67,15 +67,6 @@ contact_form:
           commands: [ "elements[0].value = elements[0].value.replace(/www\\.youtube\\.com\\/watch?[^&]*(&)?v=/gi, 'youtu.be/')" ]
           required: true
     - select:
-        - name: Your Prefix
```

Comment:
not added back as a `select`?

Reply:
Nah, it's completely missing from the form https://www.merkley.senate.gov/contact/

They might still add it back later tho, as I saw that the validation for `prefix` is still present on the page

Reply:
As long as the form submits with the update yaml, it's all good!

Reply:
oh yeah, my bad should've mentioned that. I did a test with the updated yaml and it succeeded. Thanks for checking